lib/rsvp/promise/all.js

```diff
@@ -1,22 +1,22 @@
 import Enumerator from '../enumerator';
 
 /**
-  `Promise.all` accepts an array of promises, and returns a new promise which
+  `all` accepts an array of promises, and returns a new promise which
   is fulfilled with an array of fulfillment values for the passed promises, or
   rejected with the reason of the first passed promise to be rejected. It casts all
   elements of the passed iterable to promises as it runs this algorithm.
 
   Example:
 
   ```javascript
-  import Promise, { resolve } from 'rsvp';
+  import { all, resolve } from 'rsvp';
```

Comment:
In our docs we should ensure `import { X } from 'rsvp';` should be either `Promise` or utility methods unique to RSVP. The `import { resolve, all } from 'rsvp'` are ODD, and needless different then the promise spec.

Let's use `all` from `Promise`, as that will make it easier for people to move to native.

```js
import { Promise } from 'rsvp';

Promise.all
```

Reply:
@stefanpenner your example and my changes are only about `all` but in fact your logic applies for `reject`, `resolve`, `race` and such as well, doesn't it? My idea was to make the imports consistent by importing `all` the same way as the others were already imported in those examples but I see that it would make even more sense the other way around. In fact I haven't even seen something that RSVP exports which is unique to it but I will look into it once more.  I guess in any case I should update the docs to be consistent for all native Promise functions, right?